frontend/coprs_frontend/coprs/logic/builds_logic.py

```diff
@@ -1350,9 +1350,12 @@ def new(cls, build, mock_chroot, **kwargs):
         Each freshly created instance of BuildChroot has to be assigned to
         pre-existing Build and MockChroot, hence the mandatory arguments.
         """
-        copr_chroot = coprs_logic.CoprChrootsLogic.get_by_mock_chroot_id(
-            build.copr, mock_chroot.id
-        ).one()
+        copr_chroot = kwargs.pop("copr_chroot", None)
```

Comment:
It probably doesn't matter, but `.pop` will modify the dict, and I think it is not the intention here. Maybe use `.get("copr_chroot")` instead?

Reply:
We use the **kwargs later, so I'm afraid that `copr_chroot` - if not popped - would be specified twice